examples/general-plugins/ProfileOutput/CylinderProfileDrop/plot.py

```diff
@@ -0,0 +1,44 @@
+import matplotlib.pyplot as plt
+import numpy as np
+

```

Comment:
Put in a 3-4 lines of comments here: What the script does, what the user may need to change to get it running himself/herself.

Reply:
added documentation 22de556